bundle/compliance/cis_aws/rules/cis_1_16/rule.rego

```diff
@@ -0,0 +1,47 @@
+package compliance.cis_aws.rules.cis_1_16
```

Comment:
I hope you enjoyed learning Rego 😄 
I wonder if we can write the rule in a more concise way.
We can use `in` | `if` | `some` to help us here.

maybe something like this:
```
policy_is_permissive if {
	some statement in data_adapter.policy_document.Statement
	statement.Effect == "Allow"
	statement.Action == "*"
	statement.Resource == "*"
}
```

wdyt?

Reply:
The problem I found is that both `.Action` and `.Resource` can be either strings or arrays, meaning that the above rule will have to be repeated 2x2 times. Any way to improve on that?

Reply:
Got it, Maybe in the data adapter we can check if `Action` and `Resource` are arrays, and if not create one containing this string.
Not completely sure it's possible in Rego, but if so it can simplify the rule's implementation.

Reply:
I think I managed to make it more concise with a new helper function. Take a look.